feat: suggestion for an about page briefly describing the geostyler ecosystem

OSHistory opened the pull request:
A text to be linked from the main page with a "Read More" button. It aims to shortly describe the different tools 

Before merging it should 

 - [ ] be reviewed by some core members if the ecosystem is described correctly.  
 - [ ] be reviewed by a native speaker (sometimes my syntax is way to German
 - [ ] get Christian Mayers consent that the example for the API has a POST Request to the meggsimum hosted API

geographika commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+internally the Geostyler style format. Each style parser is provided as seperate Github 
+repository and NPM package. You can get an overview of the available style parsers and 
+links [here](/parsers). Be aware that not every styling capability of a particular style 
+is supported. Lucky for you, Geostyler is open source and it is easy to contribute and get 
```
Lucky -> Luckily

jansule replied:
Should it be

```
Luckily for you, ...
```

or

```
Luckily, ...
```

geographika commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+We hope that Geostyler can help you building open geospatial tools 
+and maps. If it does we invite you to get involved: Whether you 
+are a developer providing bug fixes and features or a user improving 
+the documentation or writing tutorials. We wish you happy and stylisch 
```
stylish

geographika reviewed (approved):
Reads very well! A few minor corrections.

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+
+One key project of the ecosystem is the React Library providing developers with widgets and 
+tools to build powerfull and flexible UIs to style geodata over the web. To get an idea of 
+it's features and capabilities you can have a look at the [Demo Application](https://geostyler.github.io/geostyler-demo/) or dive into the [Source Code](https://github.com/geostyler/geostyler-demo). If you want to to learn how to build your own custom map styling applications 
```
```suggestion
it's features and capabilities you can have a look at the [Demo Application](https://geostyler.github.io/geostyler-demo/) or dive into the [Source Code](https://github.com/geostyler/geostyler-demo). If you want to learn how to build your own custom map styling applications 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+Geostyler supports multiple style formats used in the open source geospatial ecosystem: 
+QML from QGIS, Mapbox JSON used in MapLibre, OpenLayers Styling Format and the OGC-standardized 
+SLD format used by Geoserver. Each of those can be transformed into one another using 
+internally the Geostyler style format. Each style parser is provided as seperate Github 
```
I'm not sure, if we should mention our internal format, since we do not really want to give the impression that we established "just another standard". So this might be a little too technical information. Maybe something like

```
Each of those can be transformed into one another.
```

Is already sufficient.

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+rendering engine everyone (and particular your boss) is talking about. The Geostyler ecosystem is 
+here to help. 
+
+###  React Library
```
Maybe we should add a sentence that emphasizes that the UI can be used with any supported styling format, as this is the key selling point and main advantage in comparison to other UIs

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+repository and NPM package. You can get an overview of the available style parsers and 
+links [here](/parsers). Be aware that not every styling capability of a particular style 
+is supported. Lucky for you, Geostyler is open source and it is easy to contribute and get 
+involved in the community. Styles can either be transformed using javascript in your application 
```
```suggestion
involved in the community. Styles can either be transformed using JavaScript in your application 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+links [here](/parsers). Be aware that not every styling capability of a particular style 
+is supported. Lucky for you, Geostyler is open source and it is easy to contribute and get 
+involved in the community. Styles can either be transformed using javascript in your application 
+or with the Geostyler client (CLI). 
```
```suggestion
or with the Geostyler command-line interface (CLI). 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+
+### Batch Conversion with the Geostyler CLI 
+
+The Geostyler client let's you convert single style files or entire directories from one of the 
```
```suggestion
The Geostyler command-line interface let's you convert single style files or entire directories from one of the 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+### Batch Conversion with the Geostyler CLI 
+
+The Geostyler client let's you convert single style files or entire directories from one of the 
+supported styles into another. Assuming you have node installed on your system, installation 
```
```suggestion
supported formats into another. Assuming you have node installed on your system, installation 
```

jansule replied:
```
Assuming you have node installed on your system
```

I think this info might be obsolete with the latest changes @geographika did on the geostyler-cli repo :grinning:

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+
+The Geostyler client let's you convert single style files or entire directories from one of the 
+supported styles into another. Assuming you have node installed on your system, installation 
+is as easy as typing `npm install -g geostyler-cli`. Once you have the client installed you 
```
```suggestion
is as easy as typing `npm install -g geostyler-cli`. Once you have the CLI installed you 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+The Geostyler client let's you convert single style files or entire directories from one of the 
+supported styles into another. Assuming you have node installed on your system, installation 
+is as easy as typing `npm install -g geostyler-cli`. Once you have the client installed you 
+can transform a style from QGIS QML format to SLD directly from the command line: 
```
```suggestion
can transform a style, e.g. from QGIS QML format to SLD directly from the command-line: 
```

jansule commented on content/about/index.md:
````diff
@@ -0,0 +1,122 @@
+geostyler -s qgis -t sld -o output.sld input.qml
+```
+
+Be sure to have a look at the [repository](https://github.com/geostyler/geostyler-cli) for 
````
```suggestion
Make sure to have a look at the [repository](https://github.com/geostyler/geostyler-cli) for 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+
+### Conversion over the Web
+
+Maybe for your setup and workflow a conversion over the web via HTTP requests 
+is the way to go. If so, be sure to have a look at the Geostyler API. Using GET and POST 
```
```suggestion
Maybe a conversion over the web via HTTP requests is the way to go for your setup and workflow.
If so, make sure to have a look at the Geostyler REST API. Using GET and POST 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+Maybe for your setup and workflow a conversion over the web via HTTP requests 
+is the way to go. If so, be sure to have a look at the Geostyler API. Using GET and POST 
+requests you can transform your style over the web. The REST API is built on top of the popular 
+nodeJS framework express. Check out the [documentation](https://services.meggsimum.de/geostyler-rest/api-docs) and the [repository](https://github.com/geostyler/geostyler-rest). 
```
```suggestion
nodeJS framework `express`. Check out the [documentation](https://services.meggsimum.de/geostyler-rest/api-docs) and the [repository](https://github.com/geostyler/geostyler-rest). 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+is the way to go. If so, be sure to have a look at the Geostyler API. Using GET and POST 
+requests you can transform your style over the web. The REST API is built on top of the popular 
+nodeJS framework express. Check out the [documentation](https://services.meggsimum.de/geostyler-rest/api-docs) and the [repository](https://github.com/geostyler/geostyler-rest). 
+Similar to the example above the following curl call will transform a simple Mapbox (JSON-Format) style to SLD (XML): 
```
```suggestion
Similar to the example above, the following cURL call will transform a simple Mapbox (JSON-Format) style to SLD (XML): 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+### Invitation 
+
+We hope that Geostyler can help you building open geospatial tools 
+and maps. If it does we invite you to get involved: Whether you 
```
```suggestion
and maps. If it does, we invite you to get involved: Whether you 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+
+We hope that Geostyler can help you building open geospatial tools 
+and maps. If it does we invite you to get involved: Whether you 
+are a developer providing bug fixes and features or a user improving 
```
```suggestion
are a developer providing bug fixes and features, or a user improving 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,122 @@
+the documentation or writing tutorials. We wish you happy and stylisch 
+mapping.   
+
+
+
+
```
These can probably be removed, right?

jansule reviewed (changes requested):
Great work @OSHistory. I just made some points regarding the phrasing here and there. I really like the general idea of this!

jansule commented:
@chrismayer would it be ok, if we link to your demo instance of the GeoStyler REST API on the homepage?

marcjansen commented on content/about/index.md:
```diff
@@ -28,30 +28,30 @@ internally the Geostyler style format. Each style parser is provided as seperate
-supported styles into another. Assuming you have node installed on your system, installation 
-is as easy as typing `npm install -g geostyler-cli`. Once you have the client installed you 
-can transform a style from QGIS QML format to SLD directly from the command line: 
+The Geostyler command-line interface let's you convert single style files or entire directories from one of the 
```
```suggestion
The GeoStyler command-line interface let's you convert single style files or entire directories from one of the 
```

marcjansen commented on content/about/index.md:
```diff
@@ -28,30 +28,30 @@ internally the Geostyler style format. Each style parser is provided as seperate
 requests you can transform your style over the web. The REST API is built on top of the popular 
-nodeJS framework express. Check out the [documentation](https://services.meggsimum.de/geostyler-rest/api-docs) and the [repository](https://github.com/geostyler/geostyler-rest). 
-Similar to the example above the following curl call will transform a simple Mapbox (JSON-Format) style to SLD (XML): 
+nodeJS framework `express`. Check out the [documentation](https://services.meggsimum.de/geostyler-rest/api-docs) and the [repository](https://github.com/geostyler/geostyler-rest). 
```
```suggestion
Node.js framework `express`. Check out the [documentation](https://services.meggsimum.de/geostyler-rest/api-docs) and the [repository](https://github.com/geostyler/geostyler-rest). 
```

OSHistory commented:
Thanks for the review. I added your suggestions and also removed the unnecessarily technical response from the API call. 

@jansule  I concurr that there should be added a sentence clarifying at then end of the first part, but I am not ffeling to 
prosaic right now :) 

https://github.com/geostyler/geostyler-homepage/pull/12#discussion_r1192051961

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+fragment = "content"
+weight = 10
+background = "light"
+title = "About Geostyler"
```
```suggestion
title = "About GeoStyler"
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+title = "About Geostyler"
++++
+
+Geostyler is about styling maps. Maybe you are a developer looking for a set of UI components 
```
```suggestion
GeoStyler is about styling maps. Maybe you are a developer looking for a set of UI components 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+to build custom and flexible styling widgets for your geospatial application. Maybe you 
+are a cartographer trying to get the style you worked so hard on in your favorite (Q)GIS into the 
+web. Or maybe you are a project lead worrying about migrating all your styles to that new fancy 
+rendering engine everyone (and in particular your boss) is talking about. The Geostyler ecosystem is 
```
```suggestion
rendering engine everyone (and in particular your boss) is talking about. The GeoStyler ecosystem is 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+
+### Style Parsers and Transformations
+
+Geostyler supports multiple style formats used in the open source geospatial ecosystem: 
```
```suggestion
GeoStyler supports multiple style formats used in the open source geospatial ecosystem: 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+SLD format used by Geoserver. Each of those can be transformed into one of the others. Each style parser is provided as seperate Github 
+repository and NPM package. You can get an overview of the available style parsers and 
+links [here](/parsers). Be aware that not every styling capability of a particular style 
+is supported. Luckily, Geostyler is open source and it is easy to contribute and get 
```
```suggestion
is supported. Luckily, GeoStyler is open source and it is easy to contribute and get 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+links [here](/parsers). Be aware that not every styling capability of a particular style 
+is supported. Luckily, Geostyler is open source and it is easy to contribute and get 
+involved in the community. Styles can either be transformed using JavaScript in your application 
+or with the Geostyler command-line interface (CLI). 
```
```suggestion
or with the GeoStyler command-line interface (CLI). 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+### Conversion over the Web
+
+Maybe a conversion over the web via HTTP requests is the way to go for your setup and workflow.
+If so, make sure to have a look at the Geostyler REST API. Using GET and POST 
```
```suggestion
If so, make sure to have a look at the GeoStyler REST API. Using GET and POST 
```

jansule replied:
```suggestion
If so, make sure to have a look at the GeoStyler REST API. Using GET and POST 
```

jansule commented on content/about/index.md:
```diff
@@ -0,0 +1,87 @@
+ 
+### Invitation 
+
+We hope that Geostyler can help you building open geospatial tools 
```
```suggestion
We hope that GeoStyler can help you building open geospatial tools 
```

jansule reviewed (approved):
Thanks for addressing all the remarks and sorry for the late response. This will be merged now

jansule commented:
Content is online :rocket: 

https://geostyler.org/about/
